Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hmcts/cnp-java-base:openjdk-8u191-jre-alpine3.9-2.0.1
+FROM hmcts/cnp-java-base:openjdk-8-distroless-1.0
```

Comment:
don't you need to add the app insights jar if you're using distroless? as there's no curl in the image to download it at build time

Reply:
well spotted